### PR TITLE
Feature/html css files

### LINKED ISSUE #10: Choose a layout
_[idea 1, Blue and white](https://dribbble.com/shots/16026216-Project-Management-Platform/attachments/7868188?mode=media)_
- [x] [idea 2, more colors green,purple](https://dribbble.com/shots/15978828-Company-Perfomance-Dashboard/attachments/7815661?mode=media)

 @Davidschlundtbodien please add a comment about the colors you prefer

### LINKED ISSUE #12: While developing, run `npm start`.
- [x]  While developing, run `npm start`.t

### LINKED ISSUE #22: Add charts - Css in progress
- [x] add charts to Hydration 
- [x] add img to Hydration 
- [x] add charts to Steps 
- [x] add image to Steps 
- [x] add chart to friends  area.

### LINKED ISSUE #24: Color- Css in progress
- [x] choose and add to CSS color for the borders
- [x] choose and add to CSS color for the fonts
- [x] choose and add to CSS color webpage ( background color etc)

### LINKED ISSUE #27: Add Calendar
- [x] add calendar library ( dependency)
- [x] css for this calendar

### PR DESCRIPTION
**Pull request by joanafbrito**
# Description
- I've add a HTML and CSS files, created a user-charts.js file to add the charts to the page and imported them into the scripts.js 
- I've added 2 dependencies charts.js and js-datepicker.
- To create and test I've used 2 sizes : 1366x768 screen and iPhone X.
- add all the images inside the images and imported them in the scripts.js file 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?
- [x] open index.html
- [x] console.log()
- [x] dev tools
- [ ] npm test

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/css/styles.css [hunk 1]
```diff
@@ -1,5 +1,253 @@
...
+}
+
+.user-info > span {
+  right: 20px;
+  top: 50%;
+  position: absolute;
+  transform: translateY(-50%);
+}
+
+.user-info >span #calendar-input {
```
Comment (Davidschlundtbodien) [inline src/css/styles.css]: This selector (and some others below) needs a space between  the ">" and "span" I believe.

File: src/data/user-charts.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+import Chart from 'chart.js/auto'; 
+
+const sleepChart = document.getElementById('sleep-chart');
+const hydrationChart = document.getElementById('hydration-chart');
+const stepsChart = document.getElementById('steps-chart');
+const friendsChart = document.getElementById('friends-avarage-chart');
+// I think this one maybe let not const.
+const friendlabels = ['friend1', 'friend2', 'friend3','friend4','friend5'];
+
+console.log("blabvlabalba");
```
Comment (Davidschlundtbodien) [inline src/data/user-charts.js]: Awesome console message! Not sure if this was left in be accident or we will use this later.

File: src/scripts.js [hunk 3]
```diff
@@ -4,14 +4,28 @@
...
+
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
-import './images/turing-logo.png'
+import './images/Fitlit_logo.svg';
+import './images/calendar.svg';
+import './images/friends_icon.png';
+import './images/cup.png';
+import './images/foot_icon.svg';
+import './images/moon_weather.svg';
+
```
Comment (Davidschlundtbodien) [inline src/scripts.js:15]: Awesome list of new Images we can use!

**Review by Davidschlundtbodien — CHANGES_REQUESTED** [review]
Just formatting of the styles CSS (left in a line comment) otherwise the rest looks good.

File: src/data/user-charts.js (hunk 2 above)
Comment (joanafbrito) [reply src/data/user-charts.js]: Good catch! I'm still working in this file, thank you, I will delete the console.log

File: src/css/styles.css (hunk 1 above)
Comment (joanafbrito) [reply src/css/styles.css]: Just finished those spaces :)

File: src/data/user-charts.js (hunk 2 above)
Comment (joanafbrito) [reply src/data/user-charts.js]: Deleted!

**Pull request merged**